prow/plugins/verify-owners/verify-owners_test.go

```diff
@@ -292,3 +294,134 @@ func TestHandle(t *testing.T) {
 		}
 	}
 }
+
+func TestParseOwnersFile(t *testing.T) {
+	cases := []struct {
+		name  string
+		patch string
+		err   bool
```

Comment:
nit: given that nearly all scenarios have an error, I'd switch this to pass bool

Reply:
actually I can use errLine = 0 instead